packages/material-ui-unstyled/src/ModalUnstyled/ModalUnstyled.js

```diff
@@ -274,7 +274,6 @@ const ModalUnstyled = React.forwardRef(function ModalUnstyled(props, ref) {
           disableAutoFocus={disableAutoFocus}
           disableRestoreFocus={disableRestoreFocus}
           getDoc={getDoc}
-          isEnabled={isTopModal}
```

Comment:
It might be needed after all 😁

Reply:
I tried to break the nested modal demo but it's still working. Since `document.addEventListener` is a FIFO the last listener is always the top modal, so it bounces the focus correctly.

Reply:
Oh, interesting. My assumption was wrong. I forgot about the need for this prop when multiple modals are mounted at once, but not nested, only stacked. Each tries to steal the focus away.